terraform/staging/main.tf

```diff
@@ -31,6 +31,10 @@ data "terraform_remote_state" "common_infra" {
 locals {
   environment                    = "staging"
   cloudwatch_log_expiration_days = 30
+  dclg_access_group_notification_settings = {
+    enabled                     = true
+    additional_recipient_emails = ["Group-DLUHCDeltaDevNotifications+staging@softwire.com"]
```

Comment:
Should start including `delta-notifications@levellingup.gov.uk` for stuff like this